### PR TITLE
goType helper can now correctly detect arrays of built-in types

### PR DESCRIPTION
**Pull request by vgheri**

### CODE REVIEW

File: helpers.go [hunk 1]
```diff
@@ -112,27 +112,54 @@ func isFieldRepeated(f *descriptor.FieldDescriptorProto) bool {
 func goType(pkg string, f *descriptor.FieldDescriptorProto) string {
 	switch *f.Type {
 	case descriptor.FieldDescriptorProto_TYPE_DOUBLE:
+		if *f.Label == descriptor.FieldDescriptorProto_LABEL_REPEATED {
+			return fmt.Sprintf("[]float64")
```
Comment (QuentinPerez) [inline helpers.go]: You don't need to call `Sprintf` instead you can just return "[]float64"

Comment (vgheri) [inline helpers.go]: Ops you're right, copy/paste from existing code block. My bad.

**Comment by vgheri** [comment]
Removed useless fmt.Sprintf statements

**Comment by vgheri** [comment]
@moul Could you please review this PR? Thanks

**Pull request merged**